plat/nxp/common/aarch64/nxp_helpers.S

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright 2018-2019 NXP
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ */
+
+#include <asm_macros.S>
+#include <drivers/console.h>
+#include <platform_def.h>
+
+
+	.globl	plat_crash_console_init
+	.globl	plat_crash_console_putc
+	.globl	plat_crash_console_flush
+	.globl  plat_core_pos
+	.globl  plat_my_core_pos
+	.globl  plat_core_mask
+	.globl  plat_my_core_mask
+	.globl  plat_core_pos_by_mpidr
+	.globl _disable_ldstr_pfetch_A53
+	.globl _disable_ldstr_pfetch_A72
+
+#define CPUACTLR_EL1		S3_1_C15_C2_0
+#define CPUACTLR_L1PCTL_MASK	0x0000E000
+#define CPUACTLR_DIS_LS_HW_PRE	0x100000000000000
```

Comment:
This should be in the appropriate CPU header, and you should include the CPU header here.

Reply:
This is the only open comment. I am working on this. Meanwhile I have raised the pull request which takes care of other comments. Please have a look.